### PR TITLE
feat: Support seek subscription in AdminClient

### PR DESCRIPTION
**Pull request by tmdiep**
Seek subscription performs an out-of-band seek for a subscription to a specified target, which may be a backlog location, publish timestamp or event timestamp.

### CODE REVIEW

**Comment by tmdiep** [comment]
Note: this feature is pre-release. The "do not merge" label will be removed once publicly available.

File: samples/snippets/seek_lite_subscription_example.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    try:
+        seek_operation = client.seek_subscription(subscription_path, seek_target)
+        print(f"Seek operation: {seek_operation.operation.name}")
+    except NotFound:
+        print(f"{subscription_path} not found.")
+        return
+
+    # Optional: Wait for the seek operation to complete, which indicates when
+    # subscribers for all partitions are receiving messages from the seek
+    # target.
```
Comment (anguillanneuf) [inline samples/snippets/seek_lite_subscription_example.py:52]: Would you mind move some of the comment to where users set `seek_target`?

Comment (tmdiep) [reply samples/snippets/seek_lite_subscription_example.py]: I added the comments just below the `TODO(developer)` block.

File: google/cloud/pubsublite/internal/wire/admin_client_impl.py [hunk 2]
```diff
@@ -105,6 +112,29 @@ def update_subscription(
...
         )
 
+    def seek_subscription(
+        self,
+        subscription_path: SubscriptionPath,
+        target: Union[BacklogLocation, PublishTime, EventTime],
+    ) -> Operation:
+        request = SeekSubscriptionRequest(name=str(subscription_path))
+        if isinstance(target, PublishTime):
+            ts = Timestamp()
```
Comment (dpcollins-google) [inline google/cloud/pubsublite/internal/wire/admin_client_impl.py:122]: this munging is actually not necessary. The proto-plus-python library handles it.

**Review by dpcollins-google — APPROVED** [review]
Approved assuming fixes are made

File: google/cloud/pubsublite/internal/wire/admin_client_impl.py (hunk 2 above)
Comment (tmdiep) [reply google/cloud/pubsublite/internal/wire/admin_client_impl.py]: Done. Thanks!

**Review by anguillanneuf — APPROVED** [review]

**Pull request merged**